integration-webcert/src/test/java/se/inera/intyg/minaintyg/integration/webcert/converter/data/CertificateDataConverterTest.java

```diff
@@ -399,6 +400,51 @@ void shallReturnErrorValueWhenValueConverterDoesntExists() {
     assertEquals(expectedCategories, actualCategories);
   }
 
+  @Test
+  void shallExcludeQuestionsOfTypeMessageFromConversion() {
+    final var expectedCategories = List.of(
+        createCertificateCategory(CAT_ONE_TEXT,
+            createCertificateQuestion(QN_ONE_TEXT, null, QN_TEXT_VALUE)
+        )
+    );
+
+    final var elements = List.of(
+        createCategoryElement(CAT_ONE_TEXT, CAT_ONE_ID, 0),
+        CertificateDataElement.builder()
+            .id(QN_ONE_ID)
+            .index(1)
+            .parent(CAT_ONE_ID)
+            .config(
+                CertificateDataConfigTextArea.builder()
+                    .text(QN_ONE_TEXT)
+                    .build()
+            )
+            .value(
+                CertificateDataTextValue.builder()
+                    .text(QN_ONE_VALUE)
+                    .build()
+            )
+            .build(),
+        CertificateDataElement.builder()
+            .id(QN_TWO_ID)
+            .index(2)
+            .parent(QN_ONE_ID)
+            .config(
+                CertificateDataConfigMessage.builder()
+                    .message(QN_ONE_TEXT)
+                    .build()
+            )
+            .build()
+    );
+
+    doReturn(expectedCategories.get(0).getQuestions().get(0).getValue())
```

Comment:
No need to mock specific behaviour if you don't need to test specific values.. so you can remove this completely if you change based on the other comments.